packages/orchestration/src/utils/start-helper.js

```diff
@@ -133,19 +133,24 @@ export const provideOrchestration = (
 
   const makeOrchestrator = pickFacet(makeOrchestratorKit, 'orchestrator');
 
-  const facade = makeOrchestrationFacade({
-    zcf,
-    zone: zones.orchestration,
-    makeRecorderKit,
-    makeOrchestrator,
-    asyncFlowTools,
-    vowTools,
-    ...remotePowers,
-  });
+  const makeOrchestratorTools = zone =>
+    makeOrchestrationFacade({
+      zone,
+      zcf,
+      makeRecorderKit,
+      makeOrchestrator,
+      asyncFlowTools,
+      vowTools,
+      ...remotePowers,
+    });
+
+  const facade = makeOrchestratorTools(zones.contract.subZone('orchestration'));
   return {
     ...facade,
+    makeOrchestratorTools,
```

Comment:
You've effectively renamed `facade` to `orchestratorTools`.
```suggestion
  const orchestratorTools = makeOrchestratorTools(zones.contract.subZone('orchestration'));
  return {
    ...orchestratorTools,
    makeOrchestratorTools,
```

Why export the made object properties and the maker?

Also, the contract zone is supposed to be reserved for the contract,
https://github.com/Agoric/agoric-sdk/blob/e5f1d9b5b7bcaee731400c36b41b945fd42e7674/packages/orchestration/src/utils/start-helper.js#L62-L65
```suggestion
  const orchestratorTools = makeOrchestratorTools(zones.orchestration);
  return {
    ...orchestratorTools,
    makeOrchestratorTools,
```


WRT naming specifically, we also discussed `makeOrchestrateKit` because it wraps `orchestrate` with other helpers. What is the rationale behind this name?

Reply:
`zones.orchestration` is the "system" part of the tree for orchestration, and user flows should not go there. It should go somewhere below `zones.contract`. The question is whether authors might realize or not that defining an orchestration flow with name `foo` uses the `foo` prefix of the root contract zone. Creating an "orchestrated flows" sub zone of the contract zone might help with that. I'm ok with either.

Reply:
I like `makeOrchestrateKit` as it's clearer it makes `orchestrate` and friends.

Reply:
I prefer it as well; I was just doing minimal renaming, but I'll do that.